Tests/SwiftCliCoreTests/ChartifyTest.swift

```diff
@@ -4,6 +4,51 @@ import Nimble
 @testable import SwiftCliCore
 
 class ChartifyFinishedTest: XCTestCase {
-    // to do
+    func testRunValidatesTheInput() throws {
+        let mock = MockInputValidator()
+        Chartify().run(userInput: "k1", inputValidator: mock, chartConstructor: ChartConstructor())
+        let result = mock.wasValidateCalled
+        let expectedResult = true
+    expect(result).to(equal(expectedResult))
+}
+
+    func testRunTransformsTheInputIntoAnArrayIfInputIsValid() throws {
+        let mock = MockInputValidator()
+        Chartify().run(userInput: "k1", inputValidator: mock, chartConstructor: ChartConstructor())
+        let result = mock.wasArrayMakerCalled
+        let expectedResult = true
+        expect(result).to(equal(expectedResult))
+    }
+
+    func testRunMakesTheChartIfInputIsValid() throws {
```

Comment:
This is me being somewhat pedantic, but what you're testing is not whether `Chartify.run()` makes the chart if input is valid, but rather that it calls the `ChartConstructor.makeChart()` method. I would be more precise with all three of these test names.